.gitignore

```diff
@@ -18,3 +18,6 @@ packages/**/generated/**
 !integration-tests/data/standalone-0/keys/privKey
 integration-tests/data/standalone-*/
 integration-tests/data/bootstrap-0/p2p/
+
+core/ordersync/dht
```

Comment:
Can you change the tests so that these directories aren't created in the first place? Should be as simple as changing `DATA_DIR`. Let's move things to the __/tmp__ directory if possible.

Reply:
Yes, I can do that. I originally tried to use `/dev/null`, but `/tmp` would actually work!